projects/npm-tools/packages/npm-scripts/src/scripts/build.js

```diff
@@ -94,18 +94,31 @@ module.exports = async function (...args) {
 		buildSoy();
 	}
 
-	runBabel(
-		BUILD_CONFIG.input,
-		'--out-dir',
-		BUILD_CONFIG.output,
-		'--source-maps'
-	);
+	const disableOldBuild =
+		FEDERATION_CONFIG && FEDERATION_CONFIG.disableOldBuild;
```

Comment:
I couldn't find a better name. However, this should be temporary, so we don't need to be very careful with its definition, IMO.

Reply:
![judging-you](https://user-images.githubusercontent.com/7074/101782036-d54fa880-3af8-11eb-8d4b-176d297687f8.gif)

Reply:
![I don't care](https://media.giphy.com/media/SrgzeJQVnBYyI/giphy.gif)

Reply:
<img width="260" alt="Screen Shot 2020-12-10 at 15 18 05" src="https://user-images.githubusercontent.com/905006/101790973-eef1ef80-3afa-11eb-9b11-4ec39ebb5013.png">

Doesn't even care enough to put an image

![badass](https://user-images.githubusercontent.com/905006/101791100-0c26be00-3afb-11eb-874a-36555bbbd5ea.gif)

Reply:
It's GitHub, that doesn't embed it :sob: